boostsec/registry_validator/schema.py

```diff
@@ -5,23 +5,30 @@
 from pydantic import AnyHttpUrl, BaseModel, Field, validator
 
 
+class _ModuleBaseSchema(BaseModel):
```

Comment:
I don't really see a reason to make this private, you'll need it for the validation and upload no?